### PR TITLE
Feature 199 refining visual elements

### PR DESCRIPTION
**Pull request by Jdallos**

### CODE REVIEW

File: src/localization/fi.json [hunk 1]
```diff
@@ -221,7 +222,12 @@
         "formVersion": "Versio",
         "formStyling": "Lomakkeen ulkoasu",
         "backgroundImage": "Taustakuva",
-        "backgroundColor": "Taustaväri"
+        "backgroundColor": "Taustaväri",
```
Comment (villejuutila) [inline src/localization/fi.json:225]: These two strings could probably be removed since they're not used anymore (lines 224-225)

File: src/localization/fi.json [hunk 2]
```diff
@@ -221,7 +222,12 @@
...
+        "formVisibility": "Lomakkeen tiedot",
+        "formVisibilityLabel": "Lomakkeen näkyvyys",
+        "private": "Yksityinen (Suomi.fi)",
+        "public": "Julkinen",
+        "tooltipDescription": "Edit form title and set form to public or private."
```
Comment (villejuutila) [inline src/localization/fi.json:230]:
```suggestion
        "tooltipDescription": "Muokkaa lomakkeen otsikkoa ja näkyvyyttä."
```

File: src/localization/fi.json [hunk 3]
```diff
@@ -243,17 +249,20 @@
         "dateAndTime": "Päivämäärä & aika",
         "files": "Liitteet",
         "table": "Taulukko",
-        "submit": "Lähetä"
+        "submit": "Lähetä",
+        "tooltipDescription": "Select fields to add to form."
```
Comment (villejuutila) [inline src/localization/fi.json:253]:
```suggestion
        "tooltipDescription": "Valitse kenttiä, joita haluat lisätä lomakkeelle."
```

File: src/localization/fi.json [hunk 4]
```diff
@@ -271,7 +280,8 @@
             "minValueLabel": "Kentän pienin arvo",
             "maxValueLabel": "Kentän suurin arvo"
           }
-        }
+        },
+        "tooltipDescription": "Edit form section information. Edit field information, attributes, and display settings."
```
Comment (villejuutila) [inline src/localization/fi.json:284]:
```suggestion
        "tooltipDescription": "Muokkaa osion tai kentän tietoja ja ominaisuuksia."
```

File: src/localization/fi.json [hunk 5]
```diff
@@ -285,10 +295,7 @@
         "fieldDefiningCondition": "Ehdon määrittävä kenttä",
         "fieldConditionLabel": "Tämä kenttä näkyy, jos {0} arvo on {1}",
         "sectionConditionLabel": "Tämä osio näkyy, jos {0} arvo on {1}",
-        "formVisibility": "Lomakkeen tiedot",
-        "formVisibilityLabel": "Lomakkeen näkyvyys",
-        "private": "Yksityinen",
-        "public": "Julkinen"
+        "tooltipDescription": "Set conditions whether field is visible on form."
```
Comment (villejuutila) [inline src/localization/fi.json:298]:
```suggestion
        "tooltipDescription": "Aseta ehdollinen näkyvyys osiolle tai kentälle."
```

File: src/utils/metaform-utils.ts [hunk 6]
```diff
@@ -435,6 +435,60 @@ namespace MetaformUtils {
     return moment.duration(Math.floor(totalTime / replyCount));
   };
 
+  /**
+   * Create status section for new metaform
+   * 
+   * @returns status section for form
+   */
+  export const formStatusSection = (): MetaformField => {
```
Comment (villejuutila) [inline src/utils/metaform-utils.ts:443]:
```suggestion
  export const createFormStatusField = (): MetaformField => {
```
Not part of this PR but function names should contain verbs and also this function is returning a MetaformField, not MetaformSection.

**Review by villejuutila — CHANGES_REQUESTED** [review]
Changes requested.

File: src/localization/fi.json (hunk 1 above)
Comment (Jdallos) [reply src/localization/fi.json:225]: Done

File: src/components/editor/metaform-editor-right-drawer-feature.tsx [hunk 7]
```diff
@@ -853,6 +853,9 @@ const MetaformEditorRightDrawerFeature: FC<Props> = ({
         })
         }
       />
+      <Typography variant="body2">
+        { `${strings.draftEditorScreen.editor.features.field.fieldType}: ${field.type}` }
```
Comment (EVP1989) [inline src/components/editor/metaform-editor-right-drawer-feature.tsx:857]: field.type should be localized. Use same values that are used in left side drawer fields tab.

File: src/localization/fi.json [hunk 8]
```diff
@@ -214,15 +214,19 @@
...
         "versionInfo": "Version tiedot",
         "formVersion": "Versio",
         "formStyling": "Lomakkeen ulkoasu",
-        "backgroundImage": "Taustakuva",
-        "backgroundColor": "Taustaväri"
+        "formVisibility": "Lomakkeen tiedot",
+        "formVisibilityLabel": "Lomakkeen näkyvyys",
+        "private": "Yksityinen (Suomi.fi)",
+        "public": "Julkinen",
+        "tooltipDescription": "Muokkaa lomakkeen otsikkoa ja näkyvyyttä."
```
Comment (EVP1989) [inline src/localization/fi.json:229]:
```suggestion
        "tooltipDescription": "Lomakkeen perustiedot: otsikko, näkyvyys ja oletus-käyttäjäryhmä."
```

File: src/localization/fi.json [hunk 9]
```diff
@@ -244,17 +248,20 @@
         "dateAndTime": "Päivämäärä & aika",
         "files": "Liitteet",
         "table": "Taulukko",
-        "submit": "Lähetä"
+        "submit": "Lähetä",
+        "tooltipDescription": "Valitse kenttiä, joita haluat lisätä lomakkeelle."
```
Comment (EVP1989) [inline src/localization/fi.json:252]:
```suggestion
        "tooltipDescription": "Lisää kenttiä lomakkeelle raahaamalla"
```

**Review by EVP1989 — COMMENTED** [review]
Added comments

File: src/localization/fi.json (hunk 9 above)
Comment (villejuutila) [reply src/localization/fi.json]: Done

File: src/localization/fi.json (hunk 8 above)
Comment (villejuutila) [reply src/localization/fi.json]: Done

**Review by EVP1989 — APPROVED** [review]
Approved

**Review by villejuutila — APPROVED** [review]
